src/components/Grants/GrantsPastItem.tsx

```diff
@@ -34,9 +34,11 @@ const GrantsPastItem = ({ grant }: { grant: GrantAttributes }) => {
           {category}
         </Pill>
       </Table.Cell>
-      <Table.Cell className="GrantsPastItem__Cell GrantsPastItem__Text">
-        {Time(start_at).format('MMMM DD, YYYY')}
-      </Table.Cell>
+      {enacted_at && (
+        <Table.Cell className="GrantsPastItem__Cell GrantsPastItem__Text">
+          {Time.unix(enacted_at).format('MMMM DD, YYYY')}
```

Comment:
should we also cast the timestamp here? `Time.unix(enacted_at as number)`

Reply:
It is a number now so it isn't necessary